tape/src/main/java/com/squareup/tape/FileObjectQueue.java

```diff
@@ -138,15 +135,9 @@ public final void close() {
   @Override public void setListener(final Listener<T> listener) {
```

Comment:
Looks very sparsely used on Github https://github.com/search?utf8=%E2%9C%93&q=com.squareup.tape.FileObjectQueue+Listener&type=Code&ref=searchresults, mostly just in forks of Tape.